resources/idlharness.js

```diff
@@ -2274,6 +2285,11 @@ IdlInterface.prototype.test_interface_of = function(desc, obj, exception, expect
 IdlInterface.prototype.has_stringifier = function()
 //@{
 {
+    if (this.name == "DOMException")
+    {
+        // Stringifier is inherited from Error
```

Comment:
That would already be wrong in the case of an inherited stringifier, which we already return true for.  (But I checked the callers anyway and they're fine.)